tools/system_libs.py

```diff
@@ -451,7 +451,7 @@ def __init__(self, **kwargs):
   def get_cflags(self):
     cflags = super(MTLibrary, self).get_cflags()
     if self.is_mt:
-      cflags += ['-s', 'USE_PTHREADS=1']
+      cflags += ['-s', 'USE_PTHREADS=1', '-DUSE_THREADS']
```

Comment:
Why do we need a new macro here?

Reply:
I can't find a macro to check this, so I defined my own. @kripken is there a builtin macro to indicate `-pthread`?